.github/workflows/test.yml

```diff
@@ -104,7 +104,6 @@ jobs:
                "deb [arch=s390x] https://download.docker.com/linux/ubuntu \
                $(lsb_release -cs) stable"
             apt-get update
-            apt-cache madison docker-ce
```

Comment:
This is helpful to spot newer versions when they get added to the docker debian repository. please do not remove it.

Reply:
Added a commit for the same.